### PR TITLE
[Enhancement] use mmcv.utils.get_logger

### PR DESCRIPTION
**Pull request by AllentDan**
For #14

### CODE REVIEW

**Comment by RunningLeon** [comment]
@hhaAndroid, Hi could you take a look at this PR? Thanks a lot.

**Review by grimoire — APPROVED** [review]
LGTM

**Review by RunningLeon — DISMISSED** [review]
LGTM

**Comment by RunningLeon** [comment]
@AllentDan Could fix conflicts.

File: mmdeploy/utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+
+from mmcv.utils import get_logger
+
+
+def get_root_logger(log_file=None, log_level=logging.INFO):
```
Comment (hhaAndroid) [inline mmdeploy/utils/utils.py:7]: return type hint

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by grimoire — APPROVED** [review]

**Review by hhaAndroid — APPROVED** [review]

File: mmdeploy/backend/pplnn/wrapper.py [hunk 2]
```diff
@@ -11,6 +10,7 @@
 
 from mmdeploy.utils import Backend, parse_device_id
 from mmdeploy.utils.timer import TimeCounter
+from mmdeploy.utils.utils import get_root_logger
```
Comment (RunningLeon) [inline mmdeploy/backend/pplnn/wrapper.py:13]: `from mmdeploy.utils import get_root_logger`

File: mmdeploy/codebase/mmocr/deploy/text_detection.py [hunk 3]
```diff
@@ -12,6 +11,7 @@
 
 from mmdeploy.codebase.base import BaseTask
 from mmdeploy.utils import Task, get_input_shape
+from mmdeploy.utils.utils import get_root_logger
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmocr/deploy/text_detection.py:14]: `from mmdeploy.utils import get_root_logger`

File: mmdeploy/utils/utils.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+def get_root_logger(log_file=None, log_level=logging.INFO) -> logging.Logger:
+    """Get root logger.
+
+    Args:
+        log_file (str, optional): File path of log. Defaults to None.
+        log_level (int, optional): The level of logger.
+            Defaults to logging.INFO.
+    Returns:
+        :obj:`logging.Logger`: The obtained logger
```
Comment (RunningLeon) [inline mmdeploy/utils/utils.py:15]: could we make this docstring consistent with the codebase?

Comment (AllentDan) [reply mmdeploy/utils/utils.py:15]: It is consistent with mmdetection.... Shall we make an adaption?

Comment (RunningLeon) [reply mmdeploy/utils/utils.py:15]: We could change the format in another PR if necessary.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**